packages/micro-analytics-cli/src/handler.js

```diff
@@ -24,6 +24,13 @@ function realtimeHandler(req, res) {
   }
 }
 
+async function readMeta(req) {
+  if (await text(req)) {
+    return (await json(req)).meta;
```

Comment:
It might be bad, my thought was that it was better to return an error(not crash the server, that was bad wording) than silently fail at storing the metadata? I am fine with both so if you think we should just log the error I will change it :v: